doc/contributing.md

```diff
@@ -37,3 +46,18 @@ and then another commit should immediately follow which updates
 `.git-blame-ignore-revs`. For example:
 [1fec42d](https://github.com/matplotlib/viscm/pull/64/commits/1fec42d0baf90e00d510efd76cb6006fa0c70dc4),
 [8aa7bb0](https://github.com/matplotlib/viscm/pull/64/commits/8aa7bb01440aeca6f8bbcefe0671c28f2ce284c6).
+
+
+### Unit tests
+
+```bash
+make test
+```
+
+Unit tests require `xvfb` (X Virtual Framebuffer) to test the GUI. If `xvfb` is not
```

Comment:
Could we skip the tests instead with a warning? That's what we do on most other projects with the `pytest.importorskip` decorator (reads: import or skip)

Reply:
I'm not sure this will help, or I'm not quite understanding the suggestion :) `xvfb` isn't something we can import, it's a system dependency that needs to be present. I'll look in to other ways we might make this test conditional!

Reply:
Took some tinkering, but I figured out a good way to skip those tests: b86bf1a